src/Symfony/Component/HttpClient/Response/MockResponse.php

```diff
@@ -308,7 +308,7 @@ private static function readResponse(self $response, array $options, ResponseInt
             $offset = \strlen($body);
         }
 
-        if (isset($response->info['total_time'])) {
+        if (!isset($response->info['total_time'])) {
             $response->info['total_time'] = microtime(true) - $response->info['start_time'];
         }
```

Comment:
Just wondering why do we need this if-block twice: https://github.com/symfony/symfony/pull/40239/files#diff-7a4f5b316b06a89c94ca2309890e5c3ce140c4b97f92b1ecb6a640a8c97c259aL284-L286

Reply:
To measure the time that passes between the two calls.